# Internal message used to transmit a message from a specific topic
string source_topic
string destination_topic
string md5sum
string datatype
string message_definition
uint8[] data
